editor/components/page-attributes/order.js

```diff
@@ -2,22 +2,27 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { withInstanceId } from '@wordpress/components';
+import { withAPIData, withInstanceId } from '@wordpress/components';
 import { compose } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import PageAttributesCheck from './check';
 import { editPost } from '../../store/actions';
-import { getEditedPostAttribute } from '../../store/selectors';
+import { getCurrentPostType, getEditedPostAttribute } from '../../store/selectors';
+
+export function PageAttributesOrder( { onUpdateOrder, instanceId, order, postType } ) {
+	if ( ! get( postType, 'data.supports.page-attributes', false ) ) {
```

Comment:
We have a component for these checks `PostTypeSupportCheck`, can we reuse it here?

Reply:
Hi @youknowriad, PostTypeSupportCheck was updated in this PR to use the same condition used in core to decide when to show Post Attributes metabox: ( ! supportsPageAttributes && isEmpty( availableTemplates ).

In this case, we really want check if post type supports page attributes because it is the condition also used in the core to check when to show Order input field.